__tests__/Datepicker.js

```diff
@@ -4,26 +4,24 @@ import Calendar from '../src/Calendar'
 import { mount } from 'enzyme';
 import { format, getDaysInMonth } from 'date-fns'
 
-let datepicker
-
-beforeEach(() => {
-  datepicker = mount(<Datepicker />)
```

Comment:
Not sure what the purpose of removing this here was.

Reply:
you're right, this was unnecessary--I had previously added a test that required adding a prop to the datepicker. I'll switch it out.